prometheus-servers.retry

```diff
@@ -0,0 +1,3 @@
+10.0.1.145
+10.0.1.225
+10.0.1.240
```

Comment:
I think we dont need .retry file in repository.

Reply:
looks like need to get these into .gitignore